src/tracker-component.js

```diff
@@ -168,7 +156,7 @@ export const Tracker = (config? : Config = {}) => {
         console.log('PayPal Shopping: debug mode on.');
     }
     
-    clearExpiredCart();
+    resetExpiredCartId();
```

Comment:
If `getOrCreateValidCartId` takes care of expiring cartIds, then this is no longer necessary.

Reply:
Done.